README.md

```diff
@@ -1 +1,130 @@
+- Forward contracts would not allow for the dynamic nature of the USD liability changing with user interaction
+- Forward contracts would represent a counter-party risk
+- Standard futures contracts would settle in USD instead of bitcoin and would require more frequent adjustment of the position to track the bitcoin price like an inverse
+- Standard futures contracts have expiry that won't match the duration of the USD liability forcing appropriate expiry selection, eventual roll over, etc. of the contract
+- Inverse futures contracts only have expiry mismatch to USD liability duration
+
+This strategy avoids expiry and standard USD settled management of non-perpetuals non-inverse contracts.
+Note that given the Dealer's duty, if the USD liability it tries to hedge is non-zero, it always trade a cash-and-carry / long basis strategy.
+
+### What Does Basis Mean?
+
+In the futures market, basis represents the difference between the cash price of the commodity and the futures price of that commodity.
+
+### What Is a Cash-and-Carry Trade?
+
+A cash-and-carry trade is an arbitrage strategy that exploits the mispricing between the underlying asset and its corresponding derivative. The key to profiting from this strategy is the eventual correction in that mispricing. This strategy is commonly known as basis trading.
+
+### What is Basis Trading?
+
+In the context of futures trading, the term basis trading refers generally to those trading strategies built around the difference between the spot price of a commodity and the price of a futures contract for that same commodity. This difference, in futures trading, is referred to as the basis.
+
+### What Is Long the Basis?
+
+Long the basis is a trading strategy in which an investor who owns or has bought a commodity hedges their investment, or gives themselves a little buffer against potential market fluctuations, by selling futures contracts on that commodity. Taking such action provides a guaranteed price at which they may sell their commodities if the market price moves against their underlying position. 
+
+## The big picture
+
+Users of the bitcoin bank elect to keep a % of their account in stable USD value. The aggregate of all those sum up to a total **USD liability** for the bank. 
+
+The Dealer tracks and hedges the bank's USD liability by trading in the bitcoin derivatives market for example (different strategies are possible here).
+
+As the USD liability changes, the Dealer updates it's position to keep the balance.
+
+As the price of bitcoin in USD changes, the Dealer updates it's position to keep the balance.
+
+As the liquidity of the Dealer on the derivatives market changes due to the above two variations, the Dealer redraws funds to keep counter-party risk low or deposits more funds to keep/open larger positions.
+
+## Strategies
+
+### Perpetual Futures
+
+### Weekly/Monthly Futures
+
+### Savings account / Money Market
+
+## References
```

Comment:
those should be made with hyperlinks in context instead of stacking them at the end